src/Randomizer.SMZ3.Tracking/locations.json

```diff
@@ -305,7 +305,10 @@
       "Row": 5,
       "LocationId": 364,
       "MemoryAddress": 200,
-      "MemoryFlag": 11
+      "MemoryFlag": 11,
```

Comment:
Oof, yeah, I suppose you're right. I added it here as it's only relevant to tracker, but when building all the SMZ3 randomizer locations makes more sense from that perspective. It also solves the hex codes in JSON issue.

I'll see if I can get this in today. It'll be a pretty big change.

Reply:
@Vivelin This is now complete. Everything is now pulled into the C# code. The SM bosses is now kinda gross since they aren't really part of the original SMZ3 codebase, but it should work.